### DISCUSSION TITLE
Windows使用third_party安装ctc_decoders失败

### DISCUSSION THREAD

**Discussion by RogerLzWang** [PaddlePaddle/PaddleSpeech#3015] [Q&A]
Windows11环境下已按要求安装GCC, CMake, Make, VS (VS2022)，但是ctc_decoders的安装仍旧失败：
<img width="856" alt="image" src="https://user-images.githubusercontent.com/86905562/224229011-03af05b3-d345-416f-9598-6b4724f997b3.png">
<img width="497" alt="image" src="https://user-images.githubusercontent.com/86905562/224229110-26ccbead-5aa7-4475-9a54-5cbf0f91a586.png">
<img width="547" alt="image" src="https://user-images.githubusercontent.com/86905562/224229155-ebdcebdd-93d9-4a2e-a370-80cf0ba931b6.png">

运行相应bat错误如下：
[Error.txt](https://github.com/PaddlePaddle/PaddleSpeech/files/10938798/Error.txt)

**Comment by zh794390558** [comment]
看看这个 https://github.com/PaddlePaddle/PaddleSpeech/discussions/2148

**Reply by zh794390558** [reply]
scorer.cpp(6): fatal error C1083: Cannot open include file: 'unistd.h': No such file or directory
